UnityProject/Assets/Tests/Balance/CargoTest.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Text;
+using Systems.Cargo;
+using NUnit.Framework;
+using UnityEditor;
+
+namespace Tests.Balance
+{
+	public class CargoTest
+	{
+		[Test]
+		public void StonksTest()
```

Comment:
does this handle stuff with inventories and populaters?

Reply:
it does not

Reply:
you feel like adding it or shall I just merge as is?

Reply:
so check if the stuff inside the crate has inventory and then check the price for everything inside said inventory? We have a problem with recursive inventories @_@ I wished we had like a Utils.GetAllObjectInInventory to find literally everything inside something in recursive mode.

Reply:
is up to you

Reply:
I can try later. There is no rush to merge this

Reply:
ok cool

Reply:
I surrender, I don't know how to read the content of ItemStoragePopulator.